### DISCUSSION TITLE
git lfs import of a large file

### DISCUSSION THREAD

**Discussion by aminchk** [git-lfs/git-lfs#5257] [General]
Hello, I wanted to import my dataset on github but github gave me an error message saying that the file was too big. So I installed git lfs to import it.  I did the following commands:

git lfs install
cd "C:\Users\beaux\OneDrive\Bureau\DATASCIENTEST\PROJET - DATASCIENCE - NBA\sep22_cds_nba\data"
dir
git lfs track "nbashotlocations"
git add .gitattributes
git add nbashotlocations.csv
git commit -m "Add design file"
git push origin main

Then I get this error message:
![image](https://user-images.githubusercontent.com/121105876/212886652-3c05ef32-7ca3-4e8e-afad-b17936bada01.png)

Thank you.

**Comment by bk2204** [comment]
Hey,

On GitHub, Git LFS has a data quota.  The default limit is 1 GB of download bandwidth and 1 GB of storage.  This error message means you've exceeded that, and if you want to store more LFS data, you'll need to purchase a data pack.  If you have more questions about that, please contact GitHub Support, since this repository handles only the Git LFS open source project and not GitHub's implementation of it.
